features/accounts/edit/add-profile-photo/add-profile-photo.tsx

```diff
@@ -18,6 +18,8 @@ import clsx from 'clsx';
 import s from './add-profile-photo.module.scss';
 
 export const AddProfilePhoto = () => {
+  const { t } = useTranslation();
```

Comment:
This import is useless because it is used to extract data that has already been imported. Use the already imported variable `tModal`.